lottie-swift/src/Private/Model/Animation.swift

```diff
@@ -57,6 +57,13 @@ public class Animation: Codable {
   let markers: [Marker]?
   let markerMap: [String : Marker]?
   
+  /// Return all marker names, in order, or an empty list if none are specified
+  public var allMarkerNames: [String]
```

Comment:
What do you think about renaming this to just `markerNames`?

Reply:
Sure. Updated.